Dockerfile.release

```diff
@@ -11,9 +11,9 @@ RUN apt-get update && \
 # Use `1025` G/UID so users can switch between this and `heighliner` image without a need to chown the files.
 RUN groupadd -g 1025 pocket && useradd -u 1025 -g pocket -m -s /sbin/nologin pocket
 
-COPY --chown=pocket:pocket release_binaries/poktroll_linux_$TARGETARCH /bin/poktrolld
+COPY --chown=pocket:pocket release_binaries/poktroll_linux_$TARGETARCH /bin/pocketd
```

Comment:
```suggestion
COPY --chown=pocket:pocket release_binaries/pocket_linux_$TARGETARCH /bin/pocketd
```

Reply:
@red-0ne the name comes from the repo name in this case. I'm looking to change this so we also rename the release binaries.